### PR TITLE
Remove slurm-lustre v5 example and references

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: docs/vm-images.md
Comment (nick-stroud) [inline docs/vm-images.md:1]: This is problematic. 
1) If you visualize the table this change breaks the formatting.
2) The way I would read this change to the table as an outsider is that we no longer support luster with Slurm-GCP. **This is not the case**. 

I believe Slurm GCP v6 should also support Lustre with CentOS and Rocky. Do we have an example in the repo of using lustre with each os? I am thinking enterprise slurm has lustre + centos and is tested, you could point to that. I don't know that we have migrated v6 enterprise slurm example but when we do that could be the other pointer.

**Review by nick-stroud — CHANGES_REQUESTED** [review]

File: docs/vm-images.md
Comment (harshthakkar01) [reply docs/vm-images.md:1]: Updated.

We have v5 example `hpc-enterprise-slurm` using lustre and centos. So I updated it with that.
For v6 we dont currently have, I can hold this PR till we add any lustre + v6(rocky) example and then add that example here. LMKWYT

Comment (nick-stroud) [reply docs/vm-images.md:1]: I am fine to merge so long as we remember to link back to the v6 example when we create it. If you can create responsibility for that (I would put a task on a timer to remind you), then we can merge.

**Review by nick-stroud — APPROVED** [review]

File: docs/vm-images.md
Comment (harshthakkar01) [reply docs/vm-images.md:1]: I created a task and assigned it to myself. Will update it once we have example.

**Pull request merged**